src/server/api/schema.js

```diff
@@ -685,6 +693,20 @@ const rootResolvers = {
     organizations: async(_, { id }, { user }) => {
       await superAdminRequired(user)
       return r.table('organization')
+    },
+    availableActions: (_, __, { user }) => {
+      const allHandlers = process.env.ACTION_HANDLERS.split(',')
+      const availableHandlers = allHandlers.filter(handler => {
+        return require(`../action_handlers/${handler}.js`).available()
+      })
+      const availableHandlerObjects = availableHandlers.map(handler => {
+        const handlerPath = `../action_handlers/${handler}.js`
+        return {
+          'name': handler,
+          'display_name': require(handlerPath).displayName()
```

Comment:
ditto on try-...catching or just have a whitelist hardcoded of the existing ones.